packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -1004,6 +1016,10 @@ const Conversation = WebexPlugin.extend({
           actor: this.webex.internal.device.userId
         });
 
+        if (params?.isMeetingContainerActivity) {
```

Comment:
suggestion: One way to avoid having to add code here is to pass in an activity object with a prepare method. This is done for ShareActivity right now (which is quite a complex method) but it would be much more simple for the meeting container. 

I would try and avoid putting specific hooks for a specific message type in so something like `activity.dontRemoveKmsMessage` could be added in this prepare method if the activity verb was delete.

Then the code in encryption-transform just looks for that and doesn't have to check the verb or the flag for meeting container activity.

Reply:
Sounds good